frontend/src/components/CommentRect/commentRectStyle.ts

```diff
@@ -12,7 +12,7 @@ export const commentStyles = {
   width: 'max-content',
   maxWidth: '255px',
   boxSizing: 'border-box'
-}
+} as const
```

Comment:
Handy feature which makes the types become more restricted i.e. `boxSizing: 'border-box'` has type `'border-box'`, not `string`